salt/kubelet/node_labels.jinja

```diff
@@ -0,0 +1 @@
+KUBELET_NODE_LABELS="--node-labels={{ node_labels|join(",") }}"
```

Comment:
Rather than call this "node_labels", we should make it more generic.. It's an "extension" of `/etc/kubernetes/kublet` for values that are only read once upon initial startup. so `/etc/kubernetes/kublet-initial` makes sense. 

The file can then contain this for now:

    KUBELET_ARGS="--node-labels={{ node_labels|join(",") }}"

and /etc/kubernetes/kubelet should be amended to:

    source /etc/kubernetes/kubelet-initial

    # Add your own!
    KUBELET_ARGS="\
        $KUBELET_ARGS \
        ....

This provides a place for other values only read on the initial startup to be placed here too (e.g. `--register-schedulable`), prevents unnecessary service restarts since `-initial` won't have any watchers, and provides a pattern that can be used for other services with similar "read once on initial startup" variables / flags - e.g. etcd's initial clustering variables)